mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -528,6 +528,12 @@ fn get_transfers_with_empty_meta_builder<'a>(conn: &'a Connection, chain: &'a Ch
     Ok(sql_builder)
 }
 
+fn is_table_empty(conn: &Connection, table_name: &str) -> Result<bool, SqlError> {
+    let mut stmt = conn.prepare(&format!("SELECT COUNT(*) FROM {}", table_name))?;
```

Comment:
There is no need for prepared statement here (as we don't pass any dynamic argument to the query). iirc you were doing some validations on the table names somewhere before this execution (which should be improved IMO as it can be missed easily).

Reply:
One way to enhance the existing flow could be by adding a new type that ensures the safety of value by not including any of whitespace, special characters, etc. This makes it safe to pass around the query executors.